packages/firestore/src/protos/protos.json

```diff
@@ -1760,6 +1792,63 @@
                     }
                   }
                 },
+                "RunAggregationQueryRequest": {
+                  "oneofs": {
+                    "queryType": {
+                      "oneof": [
+                        "structuredAggregationQuery"
+                      ]
+                    },
+                    "consistencySelector": {
+                      "oneof": [
+                        "transaction",
+                        "newTransaction",
+                        "readTime"
+                      ]
+                    }
+                  },
+                  "fields": {
+                    "parent": {
+                      "type": "string",
+                      "id": 1,
+                      "options": {
+                        "(google.api.field_behavior)": "REQUIRED"
```

Comment:
proto3 deprecated the "required". This should be a tag only, wouldn't have any side effects, right? 🤔

Reply:
I don't know what effects it has, to be honest. But it is _definitely_ different from proto2's "required" keyword. You can just assume it's correct since it's used elsewhere in the protos.